scripts/quick-install-ubuntu.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail  ## https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+
+# This script assumes root permissions on a fresh Ubuntu Digital Ocean droplet
+# with these recommended specs: 16 GB Memory / 8 Intel vCPUs / 320 GB Disk
+
+# dismiss the popups
+export DEBIAN_FRONTEND=noninteractive 
+
+apt update -y && apt upgrade -y
```

Comment:
Needs to be run as root?

Reply:
yes, see line 4, above. Purpose of this script is getting started as quickly as possible on a fresh DO droplet